### PR TITLE
SLI-687 Show connected mode indicator with branch name

### PR DESCRIPTION
**Pull request by jblievremont**

### CODE REVIEW

File: src/main/java/icons/SonarLintIcons.java [hunk 1]
```diff
@@ -45,6 +45,8 @@
   public static final Icon WARN = IconLoader.getIcon("/images/warn.png");
   public static final Icon SCM = IconLoader.getIcon("/images/toolWindowChanges.png");
   public static final Icon PROJECT = IconLoader.getIcon("/images/ideaProject.png");
+  public static final Icon NOT_CONNECTED = IconLoader.getIcon("/images/not_connected.svg");
+  public static final Icon CONNECTED = IconLoader.getIcon("/images/connected.svg");
```
Comment (jblievremont) [inline src/main/java/icons/SonarLintIcons.java:49]: The SVG icons come from [this list](https://jetbrains.design/intellij/resources/icons_list/). I copied the ones from the "settings sync" plugin; I tried to make a version with a SonarQube or SonarCloud logo, but the icon is very small (16x16 on 1x / 96 DPI) so the logo comes out invisible.

File: src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  }
+
+  private void createPanel() {
+    layout = new CardLayout();
+    panel = new JPanel(layout);
+
+    var notConnectedCard = new JLabel(SonarLintIcons.NOT_CONNECTED);
+    var notConnectedTooltip = new TooltipWithClickableLinks.ForBrowser(notConnectedCard,
+      "<h3>Not Connected</h3>" +
+      "<p>Click to synchronize your project with SonarQube or SonarCloud.</p>" +
```
Comment (damien-urruty-sonarsource) [inline src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java:75]: Is it a click on the whole tooltip that opens the configuration panel ? I wonder if it's natural for people to click anywhere on a tooltip, maybe a link would be more explicit ?

**Review by damien-urruty-sonarsource — COMMENTED** [review]
LGTM! I hope it is clear enough that as this view is in the Current File panel, the project key and branch is specific to the file (and not to the whole project)

File: src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java (hunk 2 above)
Comment (jblievremont) [reply src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java:75]: There _is_ a link in the tooltip. `TooltipWithClickableLinks.ForBrowser` adds a default link handler that opens the link's `href` in a browser window.

Comment (damien-urruty-sonarsource) [reply src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java:75]: Are you talking about the _Learn More_ link ? I was talking about this sentence: 
> Click to synchronize your project with SonarQube or SonarCloud 

and the click listener that is set on the card

Comment (jblievremont) [reply src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java:75]: The "card" is the icon, not the tooltip.

**Pull request merged**